tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
-    py{36,37,38,39}-dj{22,31,32}
-    py{38,39}-djmain
+    py{36,37,38,39,310}-dj{22,31,32}
+    py{38,39,310}-djmain
```

Comment:
@hramezani is right, I've overseen that! Please change to 
```
py{36,37,38,39,310}-dj{22,31,32}
py310-dj{31,32}
```

Reply:
Wouldn't it be
```python
py{36,37,38,39}-dj{22,31,32}
py310-dj{31,32}
```
to exclude Django 2.2 from Python 3.10? This may be a silly question, but I'm a bit of a noob to tox, so I'm still getting acclimated to the ini config syntax.